### PR TITLE
Create genes-brains-and-behavior.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/67197/genes-brain-and-behavior-missing-citation#latest

### CODE REVIEW

File: genes-brains-and-behavior.csl [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+      <else>
+        <text variable="container-title" form="short" strip-periods="true" font-style="italic"/>
+        <text variable="collection-title" prefix=", "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " delimiter-precedes-last="never"/>
+      <label form="short" prefix=" (" suffix=")" text-case="capitalize-first"/>
```
Comment (adam3smith) [inline genes-brains-and-behavior.csl]: probably hard to find an example, but unless you have one to the contrary, make this lowercase & strip periods just like for editors in chapter citations.

File: genes-brains-and-behavior.csl [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+      <else>
+        <text variable="edition" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="locators">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper" match="any">
+        <group prefix=" " delimiter=", ">
+          <group>
```
Comment (adam3smith) [inline genes-brains-and-behavior.csl]: no need for this group

**Comment by adam3smith** [comment]
Thanks!

**Comment by rmzelle** [comment]
("brains" in the style ID should be "brain"; will fix)

**Comment by rmzelle** [comment]
Style ID fixed in https://github.com/citation-style-language/styles/pull/2928.

**Pull request merged**